src/time_integration/methods_SSP.jl

```diff
@@ -0,0 +1,188 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+
+
+abstract type SimpleAlgorithmSSP end
+
+
+"""
+    SSPRK33_ShuOsher()
+
+"""
+struct SSPRK33_ShuOsher <: SimpleAlgorithmSSP
+
+  function SSPRK33_ShuOsher()
+    new()
+  end
+end
```

Comment:
```suggestion
struct SimpleSSPRK33 <: SimpleAlgorithmSSP end
```
To fit to the naming scheme `SSPRK33` of OrdinaryDiffEq.jl?